README.md

```diff
@@ -1,54 +1,38 @@
-# rand_core
+# rand_core: core random number generation traits
 
-[![Test Status](https://github.com/rust-random/rand_core/actions/workflows/test.yml/badge.svg?event=push)](https://github.com/rust-random/rand_core/actions)
-[![Latest version](https://img.shields.io/crates/v/rand_core.svg)](https://crates.io/crates/rand_core)
-[![Book](https://img.shields.io/badge/book-master-yellow.svg)](https://rust-random.github.io/book/)
-[![API](https://docs.rs/rand_core/badge.svg)](https://docs.rs/rand_core)
+[![crate][crate-image]][crate-link]
+[![Docs][docs-image]][docs-link]
+![Apache2/MIT licensed][license-image]
```

Comment:
Unfortunately this is a clickable link to no-where. If we can't make it a non-link we should make it link to the Licence section below.